app/cafe.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:  # Added return type annotation
+        self.name: str = name

```

Comment:
Code Style: You're using type annotations, which is great! It's a good practice to use them as it makes your code more readable and maintainable.